react/components/PriceRange/PriceRangeInput.js

```diff
@@ -0,0 +1,109 @@
+import React, { useEffect, useState } from 'react'
+import { InputCurrency } from 'vtex.styleguide'
+import { useRuntime } from 'vtex.render-runtime'
+import { Button } from 'vtex.styleguide'
+import { useCssHandles } from 'vtex.css-handles'
+import { FormattedMessage } from 'react-intl'
+
+const CSS_HANDLES = [
+  'priceRangeInputWrapper',
+  'priceRangeInput',
+  'priceRangeInputButton',
+]
+
+const PriceRangeInput = ({ defaultValues, onSubmit, min, max }) => {
+  const handles = useCssHandles(CSS_HANDLES)
+  const { culture } = useRuntime()
+
+  const [defaultLeft, defaultRight] = defaultValues
+
+  const [values, setValues] = useState({
+    left: defaultLeft,
+    right: defaultRight,
+  })
+
+  useEffect(() => setValues({ left: defaultLeft, right: defaultRight }), [
+    defaultLeft,
+    defaultRight,
+  ])
+
+  const handleChange = name => e => {
+    const { value } = e.target
+
+    if (name === 'min') {
+      setValues(currentValues => ({
+        ...currentValues,
+        left: parseFloat(value),
+      }))
+    } else {
+      setValues(currentValues => ({
+        ...currentValues,
+        right: parseFloat(value),
+      }))
+    }
+  }
+
+  const handleSubmit = () => {
```

Comment:
I'm being able to submit left values bigger than the maximum value and right values smaller than the minimum value. It needs to check those cases as well.

Reply:
I didn't notice that! Now it is fixed!